Update to genotype.tabular and PR conflicts

skitchen19 opened the pull request.

skitchen19 committed:
Updated coral_mlg_rep_sample_id table.

I added a step to search the database for the previous coral_mlg_rep_sample_id for a given clonal id before adding new representative samples.

skitchen19 committed:
Call on affy_id instead of user_specimen_id

The affy_id is unique and matches the representative sample in the genotype table.

skitchen19 committed:
Update to MLG R code

Changes to the genotype table and alleles table.

skitchen19 committed:
Update to csv allele table write out

Forgot to change the name of the new allele table dataframe.

skitchen19 committed:
Comment out rep_mlg_tibble

This table is not created from the results of the database search for consistency between runs.

gregvonkuster commented:
@skitchen19 Your PR has a conflict - https://github.com/gregvonkuster/galaxy_tools/pull/5/files#diff-ac4b094dc43fdc285135f69d8deafc9eR331, so cannot be merged.  Were you changing an old version of the file?

skitchen19 committed:
Change to stag_db_report

Compares affy_id to the previously genotyped samples in the database and filters to only the new affy_id. This gets us around duplicate user_specimen_ids.

gregvonkuster commented:
@skitchen19 I'm closing this since it has conflicts and I believe everything is now handled as of this https://github.com/gregvonkuster/galaxy_tools/commit/05fdaaf623016cd84cfbc13e5bfbf4e70b25fb9c
